repo/packages/N/nexus/0/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "3.0",
+  "name": "nexus",
+  "version": "1.0-3.1.0",
+  "minDcosReleaseVersion" : "1.8",
+  "scm": "https://github.com/sonatype/docker-nexus3",
+  "maintainer": "support@sonatype.org",
```

Comment:
Adding Sonatype's email here means that Sonatype will handle updates and any questions from support team. I just want to make sure they're aware. @ssk2 would know better.